API_flask.py

```diff
@@ -0,0 +1,85 @@
+import checkmanga
+from flask import Flask, request, jsonify
+app = Flask(__name__)
+
+checkmangautil = checkmanga.CheckMangaUtil('mycheckmanga.db')
+
+def validate_input_json_keys(request_json, required_keys):
+	for key in required_keys:
+		if key not in request_json.keys():
```

Comment:
save `.keys()` somewhere so you dont have to reconstruct it in every loop

also this function is not really safe... returns either a string or a bool, which is legal only bc you're in python